src/applications/claims-status/utils/helpers.js

```diff
@@ -241,28 +241,33 @@ export function getTopPosition(elem) {
   return Math.round(box.top + scrollTop - clientTop);
 }
 
+export function stripEscapedChars(text) {
+  return text && text.replace(/\\(n|r|t)/gm, '');
+}
+
+// strip escaped html entities that have made its way into the desc
+export function stripHtml(text) {
+  return text && text.replace(/[<>]|&\w+;/g, '');
+}
+
+export function scrubDescription(text) {
+  return stripEscapedChars(stripHtml(text));
+}
+
 export function truncateDescription(text) {
   const maxLength = 120;
   if (text && text.length > maxLength) {
     return `${text.substr(0, maxLength)}…`;
   }
-  return text;
-}
-
-// strip escaped html entities that have made its way into the desc
-export function stripHtml(text) {
-  return text && text.replace(/&\w+;/g, '');
+  return scrubDescription(text);
```

Comment:
This seems like you should strip out the HTML before truncating the text, e.g. `<ThisIsMyReallyLongReactComponentNameButWillGetStrippedOut>Foo</ThisIsMyReallyLongReactComponentNameButWillGetStrippedOut>This important text may be truncated`.

Hmm, actually, I don't think the `stripHtml` function removes the text inside the `<>`

Reply:
It doesn't, what I'm usually seeing is thing like a description of `<VA Medical Facility>`. I want to keep the `VA Medical Facility` part, but want to remove the `<` and `>` characters. The EVSS version of the API returns this one as `&lt;VA Medical Facility&gt;` which is why this wasn't accounted for already